include/repeater_cadmium.hpp

```diff
@@ -0,0 +1,199 @@
+/** \brief This header file implements the Repeater class
+ * that acts as a module that takes input from subnet and
+ * after processing it send it to the receiver and get
+ * acknowledgement from receiver and forward it to sender
+ * after processing it.
+ *
+ * Repeater has 2 inputs and 2 outputs port for packet and
+ * acknowledgement respectively. It adds a 10 second
+ * processing time to packets and send it to respective output ports
+*/
+/*
+ * Peter Bliznyuk-Kvitko
+ * Shubham Agrawal
+ * Carleton University
+*/
+
+#ifndef __REPEATER_CADMIUM_H__
```

Comment:
The header guards would be #ifndef __REPEATER_CADMIUM_HPP__. Since, it is saved as .hpp, you will include to include the header guards as HPP instead of H. The same applies for other header guards in this file.

Reply:
fixed